Qt/mainwindow.cpp

```diff
@@ -25,9 +25,9 @@ MainWindow::MainWindow(QWidget *parent, bool fullscreen) :
 	QDesktopWidget *desktop = QApplication::desktop();
 	int screenNum = QProcessEnvironment::systemEnvironment().value("SDL_VIDEO_FULLSCREEN_HEAD", "0").toInt();
 	
-	// Move window to top left coordinate of selected screen
+	// Move window to the center of selected screen
 	QRect rect = desktop->screenGeometry(screenNum);
-	move(rect.topLeft());
+	move((rect.width()-frameGeometry().width()) / 4, (rect.height()-frameGeometry().height()) / 4);
```

Comment:
Hm, seems like one could also do this slightly more succinctly with:

```c++
QRect rect = frameGeometry();
rect.moveCenter(desktop->screenGeometry(screenNum).center());
move(rect.topLeft());
```

But, it's not much better.  This seems fine, although would prefer spaces around `-`.

-[Unknown]